addons/mail/__manifest__.py

```diff
@@ -154,6 +154,7 @@
             # Unused by guests and depends on ViewDialogs, better to remove it instead of pulling the whole view dependency tree
             ('remove', 'mail/static/src/components/composer_suggested_recipient/*'),
             ('remove', 'mail/static/src/components/activity_menu_container/*'),
+            ('remove', 'mail/static/src/components/avatar/*'),
```

Comment:
commit add Avatar could have a bit more explanation

Reply:
and tests?

Reply:
done